applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -21,13 +21,25 @@
 #include "bt_le_audio_tx.h"
 #include "le_audio.h"
 #include "nrf5340_audio_common.h"
+#include "bt_mgmt.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(broadcast_source, CONFIG_BROADCAST_SOURCE_LOG_LEVEL);
 
 /* Length-type-value size for channel allocation */
 #define LTV_CHAN_ALLOC_SIZE 6
 
+#if (CONFIG_AURACAST)
+/* Index values into the PBA advertising data format.
+ *
+ * See Table 4.1 of the Public Broadcast Profile, Bluetooth® Profile Specification, v1.0.
+ */
+#define PBA_UUID_INDEX		 (0)
+#define PBA_FEATURES_INDEX	 (2)
+#define PBA_METADATA_SIZE_INDEX	 (3)
+#define PBA_METADATA_START_INDEX (3)
```

Comment:
Should these two be the same? If it's already pointing to the same place it can just use the same define.

Reply:
Yes, used different naming for clarity, but I think the PBA_METADATA_START_INDEX is wrong as this should be 2 (0,1,2).
They are different so as to permit the PBA_METADATA_START_INDEX to be relocated in the buffer if need be. Constant so no additional code.